media-gfx/iscan/iscan-2.30.4.2.ebuild

```diff
@@ -0,0 +1,150 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_DOC="userg_revQ"
+inherit autotools desktop flag-o-matic
+
+# HINTS:
+# -> non-free modules are x86 and amd64 only
+# -> iscan frontend needs non-free modules
+# -> sane-epkowa should be usable on every arch
+# -> ${P}-${SRC_REV}.tar.gz    (for gcc 3.2/3.3)
+# -> ${P}-${SRC_REV}.c2.tar.gz (for gcc 3.4 or later)
+
+# FIXME:
+# iscan doesn't compile w/o libusb, this should be fixed somehow.
+
+# TODO:
+# (re)add closed-source binary modules which are needed for some scanners.
+
+DESCRIPTION="EPSON Image Scan! for Linux (including sane-epkowa backend)"
+HOMEPAGE="http://download.ebz.epson.net/dsc/search/01/search/?OSC=LX"
+SRC_URI="http://support.epson.net/linux/src/scanner/iscan/${PN}_$(ver_rs 3 -).tar.gz
+	doc? (
+		https://dev.gentoo.org/~flameeyes/avasys/${MY_DOC}_e.pdf
+		l10n_ja? ( https://dev.gentoo.org/~flameeyes/avasys/${MY_DOC}_j.pdf )
+	)"
+
+LICENSE="GPL-2 AVASYS"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc gimp l10n_ja nls X"
+
```

Comment:
You'll need to add `RESTRICT="test"` here, because it fails with `FEATURES="test"`. Try it out.

Reply:
Oops.  I guess I didn't test that.  Yes I see that doesn't work.  And as always upstream never fixes these problems.

Reply:
I restored the RESTRICT line but strangely it's still executing make check if FEATURES="test".  Is there a bug in portage?  RESTRICT="test" is supposed to block that right?

Reply:
Never mind.  I was testing the wrong ebuild.